test/domain/post/getAllAggregated.spec.ts

```diff
@@ -0,0 +1,25 @@
+
+import { beforeEach, describe, expect, it } from 'vitest';
+
+import getAllAggregated from '^/domain/post/getAllAggregated/feature';
+import { DATA_URLS, DATABASES, FILE_STORES, REQUESTERS } from './fixtures';
+
+beforeEach(async () =>
+{
+    await Promise.all([
+        DATABASES.withCreatorsPostsAndRelations(),
+        FILE_STORES.withImage()
+    ]);
+});
+
+describe('domain/post/getallAggregated', () =>
+{
+    it('should give all posts except it`s own posts for the timeline of requester1', async () =>
```

Comment:
Test descriptions should be bound to requirements, not test data. Besides that, the term `timeline` does not exist in the domain anymore.